modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -839,6 +841,63 @@ FeatureFloodCount::getFeatureVar(unsigned int feature_id) const
   return invalid_id;
 }
 
+unsigned int
+FeatureFloodCount::getFeatureID(unsigned int feature_id) const
+{   
+  if (feature_id >= _feature_id_to_local_index.size())
+    return invalid_id;
+
+  auto local_index = _feature_id_to_local_index[feature_id];
+  if (local_index != invalid_size_t)
+  {
+    mooseAssert(local_index < _feature_sets.size(), "local_index out of bounds");
+
+    return _feature_sets[local_index]._status != Status::INACTIVE
+               ? _feature_sets[local_index]._id
+               : invalid_id;
+  }
+
+  return invalid_id;
+}
+
+std::vector<unsigned int>
+FeatureFloodCount::getAdjacentID(unsigned int feature_id) const
+{ 
+  std::vector<unsigned int> invalid_id_vector(10,invalid_id);
```

Comment:
why 10?

Reply:
Oh, this is a randomly selected number. At that time, it was considered that in 2D, the data of adjacent grains of each grain rarely exceeds 10, so 10 was selected. And, the creation of this function refers to getFeatureVar.

Looking back now, it is more sensible to get set to 1